### PR TITLE
chore(core): update @angular-devkit, browserslist and caniuse-list

### PR DESCRIPTION
**Pull request by juristr**
updates the browserslist and caniuse-list packages as well as the `@angular-devkit/*` packages, `@angular/cli` and a couple of others since the same issue has also been fixed on the Angular CLI repo recently: https://github.com/angular/angular-cli/issues/16560

### CODE REVIEW

**Comment by mehrad-rafigh** [comment]
Hi @juristr. I am seeing this poping up on my terminal. Thank you for fixing it...but..You need to create a migration, so other users can enjoy your fix. Me included :)

You can refer to this [PR](https://github.com/nrwl/nx/pull/2002/files) on how to do it

**Comment by juristr** [comment]
@mehrad-rafigh 👍 thx, this is still a WIP (should have created a draft PR 👀 ). Thx for your hints though :)

File: packages/workspace/migrations.json [hunk 1]
```diff
@@ -293,6 +298,55 @@
...
+        },
+        "@angular/cli": {
+          "version": "8.3.22",
+          "alwaysAddToPackageJson": false
+        },
+        "@schematics/angular": {
+          "version": "8.3.22",
+          "alwaysAddToPackageJson": false
+        },
+        "browserslist": {
```
Comment (juristr) [inline packages/workspace/migrations.json]: I wonder whether `browserslist` and `caniuse-lite` is actually needed in the migrations 🤔

Comment (vsavkin) [reply packages/workspace/migrations.json]: They are only needed if they are present in the package.json file of the workspace. So I'd say no. 

`@angular-devkit/core` technically doesn't have to be there either.

**Review by vsavkin — CHANGES_REQUESTED** [review]

**Pull request merged**